app/src/main/java/ch/epfl/sdp/blindly/ui/main/PageViewModel.kt

```diff
@@ -7,6 +7,7 @@ import androidx.lifecycle.ViewModel
 
 class PageViewModel : ViewModel() {
 
+    //TODO Check if this part really is useful..

```

Comment:
Exactly! Not sure if this is used anywhere.

Reply:
I may have used it without completely understanding how it worked, when setting up the different tabs for the main screen with fragments...  :/
Maybe I could look into it in the next sprint, and maybe refractor the profile setup along the way?